src/pygments_tsx/tsx.py

```diff
@@ -11,8 +11,8 @@ def __init__(self, **options) -> None:
         self.tokens = TOKENS | super().tokens
 
     name = 'TypeScriptX'
-    aliases = ['tsx', 'typescriptx']
-    filenames = ['*.tsx']
+    aliases = ['tsx', 'typescriptx', 'rsx', 'toolscript']
+    filenames = ['*.tsx', '*.rsx',]
```

Comment:
should probably make a new class with a new name. If we run it this way, it'll show up in reports as `TypeScriptX` as opposed to `Toolscript`. I don't think we actually care about aliases in our use case.